packages/dds/tree/.mocharc.cjs

```diff
@@ -7,7 +7,15 @@
 
 const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");
 
-const config = getFluidTestMochaConfig(__dirname);
+// The default "spec" for this package will include `.../test/mochaHooks.js`.
+// so including it here would be redundant.
+// Additionally adding it as a required module does not work since required modules can not add hooks (see https://github.com/mochajs/mocha/issues/764).
+
+const config = getFluidTestMochaConfig(
+	__dirname,
+	[],
+	process.argv.includes("--emulateProduction") ? "PROD" : undefined,
```

Comment:
This has been confirmed to correctly report the PROD variant of the test in in the ADO test UI, see https://dev.azure.com/fluidframework/public/_build/results?buildId=358050&view=ms.vss-test-web.build-test-results-tab and search for `@Smoke`

Reply:
Can we make it so the test name also gets PROD in its title, like CJS does?

<img width="686" height="518" alt="Image" src="https://github.com/user-attachments/assets/811d5169-ff44-4343-ab0b-560d3fa970cf" />

Reply:
I think a refactor of how this labeling happens, so both are controlled by the same option, would be the right wany to do that, which seems out of scope for this PR. I agree it would be nice.